main.tf

```diff
@@ -0,0 +1,147 @@
+/**
+ * Creates a Lambda function with associated role and policies that
+ * will run the packer-janitor tool to get rid of orphaned Packer
+ * instances and their associated resources.
+ *
+ * Creates the following resources:
+ *
+ * * Lambda function
+ * * IAM role and policies to describe and delete instances, keypairs,
+ *   and security groups, as well as write messages to Cloudwatch Logs
+ * * Cloudwatch Logs group
+ * * Cloudwatch Event to regularly run cleanup job
+ *
+ * ## Usage
+ *
+ * ```hcl
+ * module "packer-janitor-lambda" {
+ *   source                 = "trussworks/lambda-packerjanitor/aws"
+ *   packer_resource_delete = "true"
+ *   packer_timelimit       = "4"
```

Comment:
Looks like this is the default. Since you don't include the other vars with defaults (cloudwatch_logs_retention_days, interval_minutes), don't know if you want to remove. Alternatively, you could set for something other than 4 and so indicate how to do non-default value.

Reply:
Okay, good call -- I've changed it in the most recent commit.